lms/templates/footer.html

```diff
@@ -83,3 +83,19 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+% if settings.FEATURES.get('ENABLE_COOKIE_CONSENT', False):
+<!-- Cookie Consent -->
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.js"></script>
+<script>
+  window.cookieconsent.initialise({
```

Comment:
I'm nervous about how we can get this to work with RTL languages. Can you put this up on a server somewhere so that I can see its behavior. You can try this yourself using the instructions here:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#coverage-testing

@marcotuts is it possible that we could release this without RTL support if it is only for European locations? Do you think the lack of support should be a blocker for merging this PR?